src/app/landing/Testimonial.css

```diff
@@ -33,3 +27,12 @@
 .Testimonial-name {
   font-style: italic;
 }
+
+.Testimonial img {
+  width: 100px;
+  border-radius: 50%;
+  border: 3px solid white;
+  -webkit-box-shadow: 0px 3px 8px 2px rgba(184,184,184,1);
```

Comment:
Doesn't our webpack config add these anyway in the build once you specify box-shadow?

Reply:
I think it should, there is autoprefixer package in the nodule_modules so I’m removing this line